### ISSUE TITLE
#enhancement: wavedrom bitfield adding `lanes` to csrconfig

### ISSUE THREAD

**Issue by iDoka** [esynr3z/corsair#15]
At this moment if I correct understand the `lanes` for wavedrom bitfield creation sliced for 16 bits per lane:
https://github.com/esynr3z/corsair/blob/master/corsair/generators.py#L104

In generic case it is good solution but in some specific cases (almost all) would be nice manually specify number of `lanes` in  `csrconfig`.

Just an example: my most adorable documentation - any datasheet from Atmel company. They use config 8 bits per lane and it is convenient way to have enough volume to print field name (even if it's not the shortest name):

![Screenshot from 2022-04-01 20-27-43](https://user-images.githubusercontent.com/4820969/161572806-9c614172-a286-4112-bfdc-b22197593076.png)

Would be nice to have such flexibility in future release 🙏🏻

**Comment by arnfol** [comment]
I accidentally found out that now corsair always generates one-line registers, which is not an expected behavior. Maybe [wavedrompy](https://github.com/wallento/wavedrompy) API changed

**Comment by esynr3z** [comment]
I agree that that would be a nice knob for documentation generation.
